packages/components/src/Chips/InternalChip.tsx

```diff
@@ -1,14 +1,9 @@
-import React, { useState } from "react";
-import classnames from "classnames";
-import styles from "./InternalChip.css";
+import React from "react";
```

Comment:
Kept this file around this time so we could use it to configure Chip easily. Really cut down on the amount of changes compared to the other branch.